Sources/PackageLoading/Diagnostics.swift

```diff
@@ -87,6 +87,39 @@ extension Diagnostic.Message {
             resource file in the resources bundle
             """)
     }
+
+    static func copyConflictWithLocalizationDirectory(path: RelativePath, targetName: String) -> Self {
+        .error("resource '\(path)' in target '\(targetName)' conflicts with other localization directories")
+    }
+
+    static func missingDevelopmentRegion() -> Self {
+        .error("missing manifest property 'developmentRegion'; it is required in the presence of localized resources")
+    }
+
+    static func localizationAmbiguity(path: RelativePath, targetName: String) -> Self {
+        .error("""
+            resource '\(path)' in target '\(targetName)' is in a localization directory and has an explicit \
+            localization declaration; choose one or the other to avoid any ambiguity
```

Comment:
would it be more clearer to say "...localization declaration in the package manifest"?

Reply:
Agreed! Will update it.